tools/build-tool/data/android/integrated-services/clientserver/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Place the service inside a subdirectory with underscores, `data/android/integrated-services/clientserver/` instead of `data/android/integrated-services/client_server/`. We use underscores to name services.